lib/Doctrine/Migrations/Configuration/Configuration.php

```diff
@@ -135,6 +149,12 @@ public function setCustomTemplate(?string $customTemplate): void
         $this->customTemplate = $customTemplate;
     }
 
+    public function setVersionComparator(string $versionComparator): void
```

Comment:
is there a need for this setter? (or functionality as part of the configurations?)

comparators are meant to be services, thus to be setted via dependency factory class... and there this already a way to provide them.

When you need external services to decide the sorting, this solution starts to be not good anymore.

(i blogged time ago on how to provide a custom comparator in https://www.goetas.com/blog/multi-namespace-migrations-with-doctrinemigrations-30/)